NuimoSDK/NuimoBluetoothController.cs

```diff
@@ -121,20 +121,23 @@ private void BluetoothGattCallback(GattCharacteristic sender, GattValueChangedEv
                 case CharacteristicsGuids.FlyCharacteristicGuidString:      nuimoGestureEvent = changedValue.ToFlyEvent();      break;
                 default:                                                    nuimoGestureEvent = null;                           break;
             }
-            if (nuimoGestureEvent != null) DispatchOnMainAsync(() => GestureEventOccurred?.Invoke(nuimoGestureEvent));
+            if (nuimoGestureEvent != null)
```

Comment:
Please make this a one-liner to be coherent with the coding style of this repo:

`if (nuimoGestureEvent != null) GestureEventOccurred?.Invoke(nuimoGestureEvent);`